src/components/atoms/MessageHeader.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <div class="flex border-b border-gray-200 px-2 py-4">
-    <img
-      :src="icons[backIcon]"
-      :alt="altText"
-      class="sm:hidden h-10 mt-auto w-10 mr-4"
-      @click="clickBack"
-      aria-label="close modal"
-      tabindex="0"
-    />
+    <button @click="clickBack" aria-label="back to inbox">
+      <img
+        :src="icons[backIcon]"
+        :alt="altText"
```

Comment:
I know you didn't create this alt text but it would be nice to say something like "back icon"

Reply:
Agreed, this is what it says on the preview right now when you click on the chat.
![image](https://user-images.githubusercontent.com/45071113/125456780-35775e82-080e-4754-aec6-7dd554a97ac0.png)